pkg/terraform/stages/baremetal/stages.go

```diff
@@ -0,0 +1,100 @@
+package baremetal
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+
+	"github.com/openshift/installer/pkg/terraform"
+	"github.com/openshift/installer/pkg/terraform/stages"
+	"github.com/openshift/installer/pkg/types"
+)
+
+// PlatformStages are the stages to run to provision the infrastructure in
+// Bare Metal.
+var PlatformStages = []terraform.Stage{
+	stages.NewStage("baremetal", "bootstrap", stages.WithNormalDestroy(), stages.WithCustomExtractHostAddresses(extractOutputHostAddresses)),
```

Comment:
The custom extract does not need to happen on the bootstrap stage. All of the outputs are in the masters stage.

Reply:
Fixed.